app/components/content/link_as_button_component.rb

```diff
@@ -0,0 +1,30 @@
+module Content
+  class LinkAsButtonComponent < ViewComponent::Base
+    attr_reader :link_text, :url, :options
+
+    include ContentHelper
+
+    def initialize(link_text:, url:, options: {})
+      super
+
+      @link_text = substitute_values(link_text)
+      @url = url
+      @options = options
+    end
+
+    def combined_options
+      allowed_keys = %i[class tabindex data id]
```

Comment:
this is good 👍 
...but as `allowed_keys` will never change when the application runs, it is probably better defined as a constant at the top of the class, e.g.
```
ALLOWED_KEYS = %i[class tabindex data id]
```

Reply:
actually ignore this comment - I think better to move the parameters as named (optional) params, e.g.

```
def initialize(text:, href:, tabindex: nil, data: {}, id: nil, class: [])
```